metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/AbstractConvertSubcommand.java

```diff
@@ -164,13 +164,13 @@ public ExitStatus execute() {
       if (destination != null) {
         if (Files.exists(destination)) {
           if (!cmdLine.hasOption(OVERWRITE_OPTION)) {
-            return ExitCode.FAIL.exitMessage(
+            return ExitCode.INPUT_ERROR.exitMessage(
```

Comment:
Minor, non-blocking. If you like the above change:

```suggestion
            return ExitCode.IO_ERROR.exitMessage(
```